files/jsondata/GroupData.json

```
@@ -2187,6 +2187,15 @@
         "XRSession: visibilitychange"
       ]
     },
+    "Window Controls Overlay API": {
+      "overview": ["Window Controls Overlay API"],
+      "interfaces": [
+        "WindowControlsOverlay"
+      ],
+      "methods": [],
+      "properties": [],
```

This could be added: 

```suggestion
      "properties": ["Navigator.windowControlsOverlay"],
```

---

@captainbrosset, please commit this.